src/articles/UnfinishedArticlePreview.js

```diff
@@ -0,0 +1,49 @@
+import { Link } from "react-router-dom";
+import * as s from "./ArticlePreview.sc";
+import {
+  secondsToMinutes,
+  secondsToHours,
+  getHumanRoundedTime,
+} from "../utils/misc/readableTime";
+export default function UnfinishedArticlePreview({ article, onArticleClick }) {
+  const handleArticleClick = () => {
+    if (onArticleClick) {
+      onArticleClick(article.id);
+    }
+  };
+
+  function titleLink(article) {
+    let linkToRedirect = `/read/article?id=${article.id}`;
+    if (article.last_reading_percentage)
+      linkToRedirect += `&percentage=${parseFloat(article.last_reading_percentage).toFixed(2)}`;
+    return (
+      <Link to={linkToRedirect} onClick={handleArticleClick}>
+        {article.title}
+      </Link>
+    );
+  }
+  function getDateString(article) {
```

Comment:
to move to `/misc/readableTime`?

Reply:
also, look into the way we're using `moment` in `ArticleSourceInfo`. Does it not do everything that we need already?

Reply:
I have used the moment library and I think that works alright :)